### ISSUE TITLE
GUI slows down with long text

### ISSUE THREAD

**Issue by endo64** [Studio3T/robomongo#1314]
GUI slows down while scrolling (on Win8.1 x64) if there is a long text without newline.
It would be nice to have a word wrap option (but I think it is not possible #225) or at least setting a max char limit to show.

If I open that collection that  has long texts Robomongo fails with below error and nothing works anymore. If I open any other collection first then opening that collection doesn't fail.

![image](https://cloud.githubusercontent.com/assets/840604/24363178/b6af8268-1317-11e7-881f-c11b1b9a1712.png)

I'm not sure if it directly related to long texts.

**Comment by simsekgokhan** [comment]
Hi @endo64 , thanks for reporting the problem.
We have tried to reproduce the problem but we could not, with some collection with long text docs.

Could you help us on how to reproduce the problem?
Maybe steps or maybe sample data from your collection or documents might be useful.
Can you also tell us your Robomongo version please?

**Comment by endo64** [comment]
Hi @simsekgokhan,
My Robomongo version 1.0.0-RC1.
I realized that this issue occurs only when I connected to a remote MongoDB instance over SSH connection.
It works without issue on local instance.

My collection is like:

```
{
    "_id" : ObjectId("587dfd03c45e4b1535d58b8d"),
    "fullname" : "User",
    "image" : "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAASABIAAD/4QBYRX...",
    "updated_at" : ISODate("2017-04-03T10:09:43.263+03:00"),
    "status" : "OK"
}
```

image is around 300KB - 1024KB base64 encoded string. Other parts of the collection is not big.

The above network problem occurs only if I use Robomongo's SSH, if I open a SSH connection using plink.exe and use that connection as a local connection in Robomongo then no network error occurs. I think I should raise another issue for that.

**Comment by simsekgokhan** [comment]
@endo64 
Thanks for the details. I did reproduce the problem and saw that UI gets slower.
We will investigate this problem.

May I know why you did not want to use GridFS? Is it because you have only one or not big image files?

**Comment by endo64** [comment]
@simsekgokhan Correct, there is only one small image that should be embedded in HTML on the application side, hence I put it inside the document.

**Comment by Fl0xer** [comment]
I have the same problem, is there some news for this issue ? I sometimes store saves of my game in my data base (a very very long string) and the UI is slowed and even frozen for 30 seconds when they appear in the results...

**Comment by endo64** [comment]
No news yet as I know. Using GridFS, URL to a file etc. are the workarounds.
